sbnanalysis/ana/SBNOsc/NueSelection.h

```diff
@@ -51,12 +52,48 @@ class NueSelection : public core::SelectionBase {
   unsigned fEventCounter;  //!< Count processed events
   unsigned fNuCount;  //!< Count selected events
 
+
+
+  TH1D* fDiffLength;
```

Comment:
Tedious though it may be, can you add documentation for all the member variables in the Doxygen format, like:
```c++
TH1D* fTrackLength;  //!< Histogram of ...
```
When you run doxygen (see [here](https://github.com/nuchicago/sbncode/tree/master/sbnanalysis#documentation)), it'll give you a list of warnings for everywhere in-code documentation is missing.